polling_stations/apps/pollingstations/helpers.py

```diff
@@ -0,0 +1,10 @@
+import re
+
+
+def format_postcode_no_space(postcode):
+    return re.sub('[^A-Z0-9]', '', postcode.upper())
+
+
+def format_postcode_with_space(postcode):
+    pc = format_postcode_no_space(postcode)
+    return pc[:-3] + ' ' + pc[-3:]
```

Comment:
The bit before the space can be 2, 3 or 4 characters. As far as I'm aware, the last bit is always 3. Here's some links that agree with me:
https://www.getthedata.com/images/postcode_format.png
https://www.mrs.org.uk/pdf/postcodeformat.pdf